drift-codec/src/main/java/io/airlift/drift/codec/metadata/ThriftEnumMetadata.java

```diff
@@ -104,14 +108,30 @@ public ThriftEnumMetadata(
 
             byEnumValue.put(value, enumConstant);
             byEnumConstant.put(enumConstant, value);
+
+            if (isThriftEnumUnknownValue(enumConstant)) {
+                checkArgument(!unknownEnumConstant.isPresent(), "Enum class %s has multiple values annotated with @ThriftEnumUnknownValue", enumClass.getName());
```

Comment:
constants -> values

This should say *constants* rather than *values* to be consistent with Java naming.